test/compilable/test19936.d

```diff
@@ -0,0 +1,17 @@
+// REQUIRED_ARGS: -de
```

Comment:
No need to add `-de` if you have a `TEST_OUTPUT`

Reply:
I don't think the output matters; it should just not do anything related to deprecations at all.

Reply:
Hm, but non-deprecation messages shouldn't matter. And deprecation messages will just cause it to error.